data.json

```diff
@@ -380,11 +380,11 @@
   {
     "name": "Hasura",
     "difficulty": "medium",
-    "description": "Submit a PR to Hasura's <a href='https://github.com/hasura/graphql-engine'>GraphQL Engine</a> or <a href='https://github.com/hasura/learn-graphql'>GraphQL Tutorial Series</a> during Hacktoberfest 2020 and get a Hasura sticker pack. For all valid PRs closing an issue with the label hacktoberfest, a secret swag will be sent.",
-    "reference": "https://hasura.io/blog/hasura-joins-hacktoberfest-3rd-year-in-a-row/",
-    "image": "https://hasura.io/blog/content/images/2020/09/Hacktoberfest-2020--5--1.png",
-    "dateAdded": "2020-10-02T12:27:00.000Z",
-    "tags": ["clothing", "hacktoberfest", "stickers", "expired"]
+    "description": "Contribute to the Hasura Docs this hacktoberfest and win t-shirt and a sticker pack if accepted.",
```

Comment:
We usually keep history by creating a new item. What do you think about this ?

Reply:
I kept the history and added a new item.